<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TablePropertySetDialog</class>
 <widget class="QDialog" name="TablePropertySetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1215</width>
    <height>811</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>775</y>
     <width>181</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="m_TableWidget">
   <property name="geometry">
    <rect>
     <x>8</x>
     <y>40</y>
     <width>1201</width>
     <height>731</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>5</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>表格宽度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_TableWidthEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>表格高度</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_TableHeightEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="m_TableSizeSetBotton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>499</x>
     <y>5</y>
     <width>281</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>第</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_CellNumberEdit"/>
    </item>
    <item>
     <widget class="QComboBox" name="m_RowColumnomboBox">
      <item>
       <property name="text">
        <string>行高</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>列宽</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="m_CellSizeEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="m_CellSizeSetBotton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>857</x>
     <y>4</y>
     <width>251</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>行数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TableRowEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>列数</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="TableColumnEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="TableSetBotton">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TablePropertySetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TablePropertySetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
